qutip/qip/gates.py

```diff
@@ -918,8 +922,8 @@ def gate_expand_3toN(U, N, controls=[0, 1], target=2):
         raise ValueError(
             "control and not target is None must be integer < integer N")
 
-    if (controls[0] == target or controls[1] == target
-            or controls[0] == controls[1]):
+    if controls[0] == target or controls[1] == target \
```

Comment:
Yes, I prefer brackets too but I was not able to break the line as a \n before * gives rise to another pep8 (\n before binary operator). I am tempted to change this into two if statements using something like 
```
if (controls[0] in {target, controls[1]} or controls[1] == target):
    ... 
```

or 

```
if (target in {controls[0], controls[1]} or (controls[0] == controls[1])):
    ...
```